app/controllers/attendances_controller.rb

```diff
@@ -0,0 +1,46 @@
+require 'awesome_print'
```

Comment:
Is autoloading but if you try to use it for debugging in the controller when a new resource is created doesn't work.
in this particular case, we forgot to remove it after debugging, it will be removed in the next pull request.